Content/Archaeology/ArchaeologyManager.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using StarlightRiver.Content.GUI;
+using StarlightRiver.Core;
+using StarlightRiver.Packets;
+using StarlightRiver.Helpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.GameInput;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.ModLoader.IO;
+using Terraria.Map;
+using Terraria.UI;
+
+namespace StarlightRiver.Content.Archaeology
+{
+    public class ArchaeologyManager : ModSystem
```

Comment:
Lets be a bit careful while naming things -- XManager is never used in any other place in the mod, ArchaeologySystem or ArchaeologyHandler would be more appropriate

Reply:
Fixed